stable/chronograf/README.md

```diff
@@ -1,4 +1,9 @@
-# Chronograf
+# DEPRECATED - Chronograf
```

Comment:
Please see https://github.com/helm/charts/blob/master/PROCESSES.md for deprecation guideline